docs/docs/10.7-update.md

```diff
@@ -8,7 +8,7 @@ next: pure-render-mixin.html
 
 React lets you use whatever style of data management you want, including mutation. However, if you can use immutable data in performance-critical parts of your application it's easy to implement a fast `shouldComponentUpdate()` method to significantly speed up your app.
 
-Dealing with immutable data in JavaScript is more difficult than in languages designed for it, like [Clojure](http://clojure.org/). However, we've provided a simple immutability helper, `update()`, that makes dealing with this type of data much easier, *without* fundamentally changing how your data is represented.
+Dealing with immutable data in JavaScript is more difficult than in languages designed for it, like [Clojure](http://clojure.org/). However, we've provided a simple immutability helper, `update()`, that makes dealing with this type of data much easier, *without* fundamentally changing how your data is represented. You can also take a look at Facebook's [Immutable-js](http://facebook.github.io/immutable-js/docs/#/) and the [Advanced Performance](http://facebook.github.io/react/docs/advanced-performance.html) section for more detail on Immutable-js.
```

Comment:
Please make links to our own docs relative from the root (`/react/docs/advanced-performance.html`). For the immutablejs link, please make it https and remove the `#/`